webpack.config.js

```diff
@@ -1,43 +1,11 @@
 const path = require( 'path' );
-const ExtractTextPlugin = require( 'extract-text-webpack-plugin' );
+const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
```

Comment:
[extract-text-webpack-plugin](https://www.npmjs.com/package/extract-text-webpack-plugin) is deprecated, and it recommends using [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin). Most of the diff in this file is due to the different syntax for that. 

I'm not sure I got the syntax right, so I'll do some regression testing on the built JS and CSS.